### PR TITLE
CIP-0019 | Adjust preamble and structure w.r.t CIP-0001

### LINKED ISSUE #663: CIP-0019 | Adjust preamble and structure w.r.t CIP-0001
Fixed by #664.

### PR DESCRIPTION
**Pull request by rphair**
Fixes #663.

([edited proposal](https://github.com/rphair/CIPs/blob/remediate-0019/CIP-0019/README.md))

### CODE REVIEW

**Comment by rphair** [comment]
@KtorZ since it seemed wrong here to move statements apparently about Rationale to where they are well situated as introductory material in the Specification (as I did in a couple other CIP remediations proceeding concurrently), this time I added just this statment in place of "Rationale: N/A" which might well be applicable to other "Informational" CIPs:
> As stated in Motivation this CIP is provided for informational purposes regarding a single deliberate design.

If there is anything else you recall from the "early days" that might serve better as Rationale going forward, then I hope you will suggest it there.

File: CIP-0019/README.md [hunk 1]
```diff
@@ -228,10 +214,31 @@ testnet:
     type-14: stake_test1uqehkck0lajq8gr28t9uxnuvgcqrc6070x3k9r8048z8y5gssrtvn
     type-15: stake_test17rphkx6acpnf78fuvxn0mkew3l0fd058hzquvz7w36x4gtcljw6kf
 ```
+## Rationale: how does this CIP achieve its goals?
+
+As stated in [Motivation](#motivation-why-is-this-cip-necessary) this CIP is provided for informational purposes regarding a single deliberate design.
+
+### Reference Implementation(s)
+
+- [input-output-hk/cardano-addresses (Byron & Shelley)](https://github.com/input-output-hk/cardano-addresses)
```
Comment (Ryun1) [inline CIP-0019/README.md:223]:
```suggestion
- [IntersectMBO/cardano-addresses (Byron & Shelley)](https://github.com/IntersectMBO/cardano-addresses)
```

File: CIP-0019/README.md [hunk 2]
```diff
@@ -228,10 +214,31 @@ testnet:
...
+
+- [input-output-hk/cardano-ledger-specs (Byron)](https://github.com/input-output-hk/cardano-ledger-specs/blob/d5eaac6c4b21a8e69dc3a5503a72e3c3bfde648e/byron/ledger/impl/src/Cardano/Chain/Common/Address.hs)
```
Comment (Ryun1) [inline CIP-0019/README.md:225]:
```suggestion
- [IntersectMBO/cardano-ledger-specs (Byron)](https://github.com/IntersectMBO/cardano-ledger-specs/blob/d5eaac6c4b21a8e69dc3a5503a72e3c3bfde648e/byron/ledger/impl/src/Cardano/Chain/Common/Address.hs)
```

File: CIP-0019/README.md [hunk 3]
```diff
@@ -228,10 +214,31 @@ testnet:
...
+
+- [input-output-hk/cardano-ledger-specs (Shelley)](https://github.com/input-output-hk/cardano-ledger-specs/blob/1e7e6e03a46e8118b318ed105214767aec0f3976/shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Address.hs)
```
Comment (Ryun1) [inline CIP-0019/README.md:227]:
```suggestion
- [IntersectMBO/cardano-ledger-specs (Shelley)](https://github.com/IntersectMBO/cardano-ledger-specs/blob/1e7e6e03a46e8118b318ed105214767aec0f3976/shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Address.hs)
```

**Review by Ryun1 — APPROVED** [review]
LGTM ⚙

File: CIP-0019/README.md [hunk 4]
```diff
@@ -228,10 +214,31 @@ testnet:
     type-14: stake_test1uqehkck0lajq8gr28t9uxnuvgcqrc6070x3k9r8048z8y5gssrtvn
     type-15: stake_test17rphkx6acpnf78fuvxn0mkew3l0fd058hzquvz7w36x4gtcljw6kf
 ```
+## Rationale: how does this CIP achieve its goals?
+
+As stated in [Motivation](#motivation-why-is-this-cip-necessary) this CIP is provided for informational purposes regarding a single deliberate design.
```
Comment (KtorZ) [inline CIP-0019/README.md:219]:
```suggestion
As stated in [Motivation](#motivation-why-is-this-cip-necessary) this CIP is provided for informational purposes regarding a single deliberate design. Further rationale and motivation for this design are available in the [Design Specification for Delegation and Incentives in Cardano - Section 3.2 :: Addresses & Credentials ](https://github.com/intersectmbo/cardano-ledger/releases/latest/download/shelley-delegation.pdf).
```

Comment (rphair) [reply CIP-0019/README.md:219]: thanks @KtorZ - nice addition

**Review by KtorZ — APPROVED** [review]

**Pull request merged**